File: test/fixtures/elixir/post_json_multiple_headers.ex
response = HTTPoison.post!(
  "http://localhost:28139/rest/login-sessions",
  "{\"userName\":\"username123\",\"password\":\"password123\", \"authLoginDomain\":\"local\"}",
  [
    {"Content-Type", "application/json"},
    {"X-API-Version", "200"}
  ],
  [hackney: [:insecure]]
)

File: test/fixtures/csharp/post_json_multiple_headers.cs
using System.Net.Http;
using System.Net.Http.Headers;

HttpClientHandler handler = new HttpClientHandler();
handler.ServerCertificateCustomValidationCallback = HttpClientHandler.DangerousAcceptAnyServerCertificateValidator;

HttpClient client = new HttpClient(handler);

HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Post, "http://localhost:28139/rest/login-sessions");

request.Headers.Add("X-API-Version", "200");

request.Content = new StringContent("{\"userName\":\"username123\",\"password\":\"password123\", \"authLoginDomain\":\"local\"}");
request.Content.Headers.ContentType = new MediaTypeHeaderValue("application/json");

HttpResponseMessage response = await client.SendAsync(request);
response.EnsureSuccessStatusCode();
string responseBody = await response.Content.ReadAsStringAsync();
